x-pack/packages/kbn-elastic-assistant-common/impl/schemas/anonymization_fields/find_anonymization_fields_route.gen.ts

```diff
@@ -17,6 +17,7 @@ import { ArrayFromString } from '@kbn/zod-helpers';
  *   version: 1
  */
 
+import { SortOrder } from '../common_attributes.gen';
```

Comment:
`SortOrder` was common across kb, conversations, and anonymization, so extracted to common attributes.